tests/core/test_test.py

```diff
@@ -40,3 +40,57 @@ async def test_testing_valid_with_non_e2e_core_model(core_agent: Agent):
         "data/test_yaml_stories/test_stories_entity_annotations.yml", core_agent
     )
     assert "report" in result.keys()
+
+
+async def test_action_unlikely_intent_warnings_1(
+    tmpdir: Path, intent_ted_policy_moodbot_agent: Agent
+):
+    result = await rasa.core.test.test(
+        "data/test_yaml_stories/test_action_unlikely_intent_1.yml",
+        intent_ted_policy_moodbot_agent,
+        out_directory=str(tmpdir),
+    )
+    assert "report" in result.keys()
+    assert result["report"]["conversation_accuracy"]["correct"] == 1
+    assert result["report"]["conversation_accuracy"]["with_warnings"] == 1
+
+
+async def test_action_unlikely_intent_warnings_2(
+    tmpdir: Path, intent_ted_policy_moodbot_agent: Agent
+):
+    result = await rasa.core.test.test(
+        "data/test_yaml_stories/test_action_unlikely_intent_2.yml",
+        intent_ted_policy_moodbot_agent,
+        out_directory=str(tmpdir),
+    )
+    assert "report" in result.keys()
+    assert result["report"]["conversation_accuracy"]["correct"] == 1
+    assert result["report"]["conversation_accuracy"]["with_warnings"] == 1
```

Comment:
Should we have a warning if `action_unlikely_intent` was in the test story?

Reply:
Well, at least that's what's written in the task description but we can try changing that.

Reply:
Done!